aten/src/ATen/core/interned_strings.h

```diff
@@ -135,6 +135,10 @@ namespace c10 {
   _(aten, wait)                    \
   _(aten, save)                    \
   _(aten, ord)                     \
+  _(aten, chr)                     \
```

Comment:
I am not sure these values are actually used. I only see them as strings not a symbols.

Reply:
Yea somehow it didn't compile without adding them here. I will double check before merging. Thanks!

Reply:
These are used as symbols in `compiler.cpp`, so it makes sense to add them here I think.